dot-base/icd-10-api
# feat: return icd-10 codesystem version

# Ticket 🎫

This closes https://github.com/dot-base/icd-10-api/issues/90

BREAKING CHANGE: changes the return type of the API search results to FuseResult<Fhir.CodeableConcept>. 🚨

# Description 📖
Provide a description of the changes. Also include motivation and context of the changes.

# How to Test? 🧪

Setup locally and request e.g. `https://dotbase.local/api/icd10/?search=parkinson`.
Check that the result items match [CodeableConcepts](https://build.fhir.org/datatypes.html#CodeableConcept)

# Is there something controversial ? 🚨
This needs to be changed in medical-dashboard too.

# Checklist ✅
